### DISCUSSION TITLE
#Task Library - Connector to ArangoDB required

### DISCUSSION THREAD

**Discussion by mahi80** [PrefectHQ/prefect#3409] [Ideas]
Hi Team,
We are working on highly scale-able implementation of ML and AI pipeline using ArangoDB not able see if it can connect to  arangoDB below link 

https://github.com/threatify/arango-orm 

Thanks
Mahendra

**Comment by joshmeek** [comment]
Hi @mahi80 the [task library](https://docs.prefect.io/core/task_library/overview.html) is mainly community driven and we highly encourage contributions if this is something you would like implemented!
